firebase-perf/src/main/java/com/google/firebase/perf/config/ConfigResolver.java

```diff
@@ -331,7 +338,12 @@ public double getNetworkRequestSamplingRate() {
       return deviceCacheValue.get();
     }
 
-    // 3. Returns default value if there is no valid value from above approaches.
+    // 3. Check if RC fetch failed.
+    if (remoteConfigManager.isLastFetchFailed()) {
```

Comment:
What is the meaning of lastFetchFailed? Are we accounting for any kind of a fetch failure to fallback to these defaults?